### ISSUE TITLE
API: /competitions/create/ returns a server error when given a participants list with length 1

### ISSUE THREAD

**Issue by kgasiorowski** [wise-old-man/wise-old-man#947]
When given the following request payload:
```
{
    "title": "Testing SOTW 74", 
    "metric": "thieving", 
    "startsAt": "2022-06-07T00:00:00.000000Z",
    "endsAt": "2022-06-14T00:00:00.000000Z", 
    "participants": [
        "Crotch Flame"
    ]
}
```

The API endpoint `/competitions/create` returns a 500 server error with the following message: `participants.filter is not a function`

If I add another user into the array, the competition is created without any other problems.

**Comment by kgasiorowski** [comment]
Came across this while testing a discord bot - not a huge deal, since a competition generally has more than one player, but I figure I'd still point it out anyway.

**Comment by psikoi** [comment]
This has been solved in #1003 closing 👋
